tests/integration/cmd_debug_test.go

```diff
@@ -119,43 +103,5 @@ var _ = Describe("odo debug command tests", func() {
 			stopChannel <- true
 		})
 
-		It("should start a debug session and run debug info on a closed debug session", func() {
```

Comment:
Similarly, as above, this should be a valid scenario.

Reply:
Tests that have been removed without a comment have a devfile counterpart already present hence can be deleted